features/step_definitions/civil_journey_steps.rb

```diff
@@ -423,6 +423,10 @@
   expect(page).to have_content('Your legal aid applications')
 end
 
+Then('I am on the Email Entry page') do
+  expect(page).to have_content('Enter your client\'s email address')
```

Comment:
I'd suggest `"Enter your client's email address"` as a little easier to read.

Reply:
done